### ISSUE TITLE
DetailsList: Accessibility - Broken Keyboard focus on Column headers when in High Contrast mode

### ISSUE THREAD

**Issue by caradong** [OfficeDev/office-ui-fabric-react#5382]
### Bug Report

- __Browser and OS versions__: windows 10 edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

#### Describe the issue:

Repro steps:
1. Turn on High Contrast Mode(Black or White) on Windows 10
2. Go to a details list 
3. Using Tab key, navigate on the column headers and check the focus visibility.

#### Actual behavior:

When in HC mode, keyboard focus is broken on table column headers. The focus is visible only on 3 sides of the button.

#### Expected behavior:

The focus to be visible on all 4 sides of the button when in HC mode like in normal mode. 

This image shows the broken keyboard focus in high contrast mode and the working fine normal mode for the same details list component

![hc focus](https://user-images.githubusercontent.com/20460495/42066290-953df574-7af4-11e8-8251-c1762610aec9.png)

**Comment by natalieethell** [comment]
Hey @caradong thanks for submitting this issue! I can't seem to repro this behavior in HC. Here's a CodePen I've made with a DetailsList: https://codepen.io/naethell/pen/QBzvrG.

I'm seeing normal behavior in both HCB and HCW. 

![image](https://user-images.githubusercontent.com/14134000/43928617-5f37d478-9be6-11e8-8995-e3b1cd02ec4f.png)

![image](https://user-images.githubusercontent.com/14134000/43928652-8443234e-9be6-11e8-83ee-8dcbf4f77819.png)

I'll go ahead and close this issue, but if you are still seeing this behavior, feel free to open the issue again.

Thanks!

**Comment by caradong** [comment]
@natalieethell I beleive someone fixed issue not long ago but I guess they did not close this issue. When I upgraded to 6.26.0, it was fixed. Thank you for looking into it.

**Comment by natalieethell** [comment]
That's great, thanks for the update!
